tests/console/redis.pm

```diff
@@ -37,7 +37,7 @@ sub run {
     validate_script_output('redis-cli get foo', sub { m/bar/ });
     validate_script_output('redis-cli pfselftest', sub { m/OK/ });
     validate_script_output('redis-cli flushdb', sub { m/OK/ });
-    validate_script_output('redis-cli get foo', sub { m/(nil)/ });
+    validate_script_output('redis-cli get foo', sub { m/(nil\n)/ });
```

Comment:
```suggestion
    validate_script_output('redis-cli get foo', sub { chomp; m/(nil)/ });
```

Reply:
https://openqa.suse.de/tests/9635022#step/redis/21 still not working, thanks!